.stylelintrc-css

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["stylelint-config-standard", "stylelint-prettier/recommended"],
-  "ignoreFiles": ["dist/less/**/*"],
+  "ignoreFiles": ["dist/less/**/*", "**/*.min.css", "src/fonts/**/*"],
```

Comment:
Why is this changed now? It was working before right?

Reply:
now that lint-staged runs on committed files, when new .min.css files are generated, stylelint gets run on them; our lint:css script doesn't run on those files

Reply:
What I'm wondering is when we run `build` even if min.css files are there stylelint isn't messing it up. Build does not clean anything so why is it necessary now.

Reply:
nuked this part